src/site/content/en/learn/design/responsive-images/index.md

```diff
@@ -164,13 +165,44 @@ For a hero image above the fold, use a `loading` value of `eager`.
 >
 ```
 
-For an important image you can tell the browser to pre-fetch the image in the head of your document.
+### Priority hints
+
+For important images-such as the (LCP)[/lcp/] image, you can further prioritise the loading using [Priority Hints](/priority-hints/) by setting the `fetchpriority` attribute to `high`:
+
+```html/5-6
+<img
+ src="hero.jpg"
+ alt="A description of the image."
+ width="1200"
+ height="800"
+ loading="eager"
+ fetchpriority="high"
+>
+```
+
+This will tell the browser to fetch the image right away, and at high priority, rather than waiting until the browsder has completed layout when images are normally fetched.
+
+But remember: when you ask the browser to prioritize downloading one resource—like an image—the browser will have to de-prioritize another resource such as a script or a font file. Only set `fetchpriority="high"` on an image if it is truly vital.
+
+### Preloading hints
+
+Some images may not be available in the initial HTML—if they are added by JavaScript, or as a [background image](#background-images) in CSS. You can also use preload to allow these important images to be fetched ahead of time. This can be combined with the `fetchpriority` attribute for really important images:
 
 ```html
-<link rel="prefetch" href="hero.jpg" as="image">
+<link rel="preload" href="hero.jpg" as="image" fetchpriority="high">
 ```
 
-But remember: when you ask the browser to prioritize downloading one resource—like an image—the browser will have to de-prioritize another resource such as a script or a font file. Only prefetch an image if it is truly vital.
+Again this should be used sparingly, to avoid overriding the browsers prioritisation heuristics too much, which may result in performance degredation.
```

Comment:
```suggestion
Again this should be used sparingly to avoid overriding the browsers prioritisation heuristics too much, which may result in performance degredation.
```